proxy/operator_resource/resource_manager.py

```diff
@@ -231,6 +288,7 @@ async def _refresh_signer_list(self) -> None:
             op_signer = self._init_op_signer(signer)
             self._deleted_holder_addr_set.update([h.address for h in op_signer.disabled_holder_list])
             self._disabled_signer_dict[signer.pubkey] = op_signer
+            await self._send_op_resource_holder_stats(op_signer)
```

Comment:
why is `await` here?
The function `_send_op_resource_holder_stats` isn't async

Reply:
Sorry, that's leftover from when I factored out ALL of the stats code from `free_resource()` which had a few `await`s in there and so `_send_op_resource_holder_stats()` also needed to be `async`. When I moved the `await`s back I didn't update the calls.

Reply:
Fixed.